exchanges/gateio/gateio_wrapper.go

```diff
@@ -112,6 +114,13 @@ func (g *Gateio) SetDefaults() {
 			Kline: kline.ExchangeCapabilitiesSupported{
 				Intervals: true,
 			},
+			FuturesCapabilities: exchange.FuturesCapabilities{
+				FundingRates:         true,
+				FundingRateFrequency: kline.EightHour.Duration(),
```

Comment:
Based on the comment I made on Binance, the funding frequency can vary from the 8-hour interval. However, your code processes it according to the interval provided in the futures contract API response. Could add another field to `FuturesCapabilities` which allows a user to know that the exchange can have dynamic intervals